BP/common/scripted_character_templates/00_invader_templates.txt

```diff
@@ -0,0 +1,42 @@
+﻿################################
```

Comment:
Why is this file being included?

Reply:
Removal of all references to magyar_pagan/magyar religion. It plays a role in hungarian invasion in vanilla CK3 and throws a number of errors if not overwritten.

Reply:
Ok